tests/test_cpython_hierarchy.py

```diff
@@ -62,3 +62,10 @@ def test_nodegraph_vs_table():
 
     assert not hasattr(x, 'consume_and_tag')
     assert hasattr(y, 'consume_and_tag')
+
+
+def test_counttable_no_unhash():
```

Comment:
I second @betatim's suggestion: `@pytest.mark.parametrize` is the way to go for testing all base and derived types.